README.md

```diff
@@ -1,12 +1,17 @@
-# RunBook
-
-Student Robotics Competition Program
+# Student Robotics Volunteer RunBook
 
 [![CircleCI](https://circleci.com/gh/srobo/runbook.svg?style=svg)](https://circleci.com/gh/srobo/runbook)
 
-# Installation
+The runbook aims to be a single source for all Student Robotics internal
+documentation. This includes information for the [Competition Team][competition-team]
+and [Kit Team][kit-team], as well as other activities.
+
+[competition-team]: https://opsmanual.studentrobotics.org/annual-robotics-competition/competition-programme-team
+[kit-team]: https://opsmanual.studentrobotics.org/v/master/annual-robotics-competition/kit-team
+
+## Installation
 
-## Requirements
+In order to bootstrap this project you'll need:
```

Comment:
Using _bootstrap_ in this context is a little strange to those who haven't seen it before, and not obvious what it means.

```suggestion
To run this project locally, you'll need:
```

Reply:
I was avoiding that phrasing as it's misleading -- not only are they not all that you need, they're also not all strictly needed anyway; instead they're the tools which let you install the actual dependencies.

Reply:
da1386d